### PR TITLE
[Frontend] Create Profile Page

### PR DESCRIPTION
**Pull request by brixtuyor**
# Related Links
- http://localhost:8000/profile/:id
- https://app.asana.com/0/1202403928837322/1202403928837336/f

# Description
- Profile page markup that displays the basic user information, followers, current lesson overview, and activities

# Notes
- Added `@reduxjs/toolkit` package, please run `npm install`
- As of the moment, the working navigations are `profile/1` and `profile/2` just to differentiate the UI between:
1. The currently logged user
2. Any other visited users

# Updates
- Added logo that's used as both favicon and brand logo
- Fixed most of the responsiveness issues

# Screenshots
image
image

### CODE REVIEW

**Comment by roseaugusto** [comment]
@brixtuyor 

1. **For this section, what will it look like if there are more than 5 followers?**

image

2. **Where can you unfollow a user (considering this is same layout with other users)? there is no button.**

**Comment by brixtuyor** [comment]
> 

1. This is the sample expected output
image

2. They can unfollow users at the Follows tab/link

**Comment by brixtuyor** [comment]
> @brixtuyor
> 
> 1. **For this section, what will it look like if there are more than 5 followers?**
> 
> image
> 
> 2. **Where can you unfollow a user (considering this is same layout with other users)? there is no button.**

Oh there should be a different route for a visited user... Wait I'll do that now.

File: resources/js/pages/profile/components/CurrentLessonOverview.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Box, Button, Flex, Heading, Text } from "@chakra-ui/react";
+
+const CurrentLessonOverview = () => {
```
Comment (roseaugusto) [inline resources/js/pages/profile/components/CurrentLessonOverview.tsx]: Use generic name or general name
```suggestion
const LessonOverview = () => {
```

File: resources/js/pages/profile/components/CurrentUserActivityList.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Flex } from "@chakra-ui/react";
+import Activity from "../../dashboard/components/ActivityCard";
+
+const CurrentUserActivityList = () => {
```
Comment (roseaugusto) [inline resources/js/pages/profile/components/CurrentUserActivityList.tsx]: Use generic name or general name
```suggestion
const UserActivityList = () => {
```

File: resources/js/pages/profile/components/CurrentLessonOverview.tsx (hunk 1 above)
Comment (brixtuyor) [reply resources/js/pages/profile/components/CurrentLessonOverview.tsx]: Done

File: resources/js/pages/profile/components/CurrentUserActivityList.tsx (hunk 2 above)
Comment (brixtuyor) [reply resources/js/pages/profile/components/CurrentUserActivityList.tsx]: Done

File: package.json [hunk 3]
```diff
@@ -16,6 +16,7 @@
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
     "@fontsource/nunito": "^4.5.9",
+    "@reduxjs/toolkit": "^1.8.2",
```
Comment (roseaugusto) [inline package.json]: @brixtuyor please exclude this for now since it's not included in this task scope and this package is not being used at the moment.

**Comment by roseaugusto** [comment]
@brixtuyor if user's own profile it's okay to used this route `http://127.0.0.1:8001/profile` put id params only if you are accessing followers profile. Currently, it's not displaying anything. (refer to the screenshot)
image

File: package.json (hunk 3 above)
Comment (brixtuyor) [reply package.json]: Removed

**Comment by brixtuyor** [comment]
> @brixtuyor if user's own profile it's okay to used this route `http://127.0.0.1:8001/profile` put id params only if you are accessing followers profile. Currently, it's not displaying anything. (refer to the screenshot) image

Fixed

**Comment by brixtuyor** [comment]
Fixed

image

**Review by roseaugusto — APPROVED** [review]
Approved!

**Pull request merged**